shoelace.js

```diff
@@ -3,13 +3,15 @@
 global.__version = require('./package.json').version;
 
 require('dotenv').config();
-const CleanCSS = require('clean-css');
 const Chalk = require('chalk');
 const Del = require('del');
 const FS = require('fs');
 const Path = require('path');
 const Program = require('commander');
 const S3 = require('s3');
+const postcss = require('postcss');
+const cssnano = require('cssnano');
+const atImport = require('postcss-import');
```

Comment:
This is nit picky but I try to name modules and classes with uppercase letters. Could we do `PostCSS`, `CSSnano`, and `AtImport` for these?